scania-digital-design-system/sdds
# fix/textfield state count issues

**Describe pull-request**  
If a user selected a char count without a state the counter would render at the wrong place. Also if the value of the textfield/textarea is null nothing would render in the char counter, this adds a "0" in that case.

**Solving issue**  
Fixes: [AB#2574](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2574)

**How to test**  
1. Checkout branch and run storybook.
2. Check in Components -> Textfield/Textarea
3. Add a char counter with and without a state and see that it renders correctly. 

**Screenshots**  
-

**Additional context**  
-
